src/modules/numbers/number-types.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable} from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { NumberType } from './entities/number-type.entity';
+
+@Injectable()
+export class NumberTypesService {
+
+  constructor(
+    @InjectRepository(NumberType, "mssql-number-generator")
+      private readonly numberTypeRepository: Repository<NumberType>,
+    ) {}
+
+
+  async findAll(): Promise<NumberType[]> {
```

Comment:
there is no `await` below, so `async` becomes redundant

Reply:
Done